components/infra-proxy-service/storage/postgres/automate_infra_server_users.go

```diff
@@ -132,3 +132,28 @@ func (p *postgres) GetAutomateInfraServerUsers(ctx context.Context, serverId str
 	}
 	return users, nil
 }
+
+func (p *postgres) GetUsers(ctx context.Context, serverID string) ([]storage.User, error) {
```

Comment:
There is already a function `GetAutomateInfraServerUsers`. Why have we added a new function here?